h2o-algos/src/main/java/hex/tree/GlobalQuantilesCalc.java

```diff
@@ -0,0 +1,46 @@
+package hex.tree;
+
+import hex.quantile.Quantile;
+import hex.quantile.QuantileModel;
+import water.DKV;
+import water.Job;
+import water.Key;
+import water.fvec.Frame;
+import water.util.ArrayUtils;
+
+public class GlobalQuantilesCalc {
```

Comment:
:+1: 

Could you add some short description to the docstring about usage, please?

Reply:
done, thank you for reminding me